src/app/pages/components/tree/tree.component.ts

```diff
@@ -29,5 +32,64 @@ export class TreeComponent {
       }],
     }],
   }];
-
+  options: ITreeOptions = {
```

Comment:
let's add an empty line between two declarations (properties, methods, etc).
the order would be 
@viewchild
public properties
private properties
constructor
public methods
private methods

Reply:
changed